windows/pre_setup.ps1

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 
-
+<powershell>
```

Comment:
this is only needed for user data, is this correct?

Reply:
Yeah. And since I was copying and pasting this into userdata, I figured it should be obvious.